lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -757,12 +779,15 @@ export class UserAgentApplication {
             }
         })
             .then(res => {
+                this.logger.verbose("Successfully acquired token");
                 this.telemetryManager.stopAndFlushApiEvent(request.correlationId, apiEvent, true);
+                this.logger.info("Stopping telemetryManager and flushing API event");
```

Comment:
I'm working in the telemetry manager now, and will add these sorts of logs there, so you can skip these here.

Reply:
Removed.